syntax = "proto3";
package protos;

option go_package = "./protos";

service HelloService {
  rpc hello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
  string name = 1;
}
message HelloResponse {
  string message = 1;
}
